at_response.c

```diff
@@ -265,6 +265,20 @@ static int at_response_ok (struct pvt* pvt, at_res_t res)
 	return 0;
 }
 
+static void log_cmd_response_error(const struct pvt* pvt, const at_queue_cmd_t *ecmd, const char *fmt, ...)
+{
+	va_list ap;
+
+	if (at_cmd_suppress_error_mode(ecmd->flags) == SUPPRESS_ERROR_ENABLED) {
+		ast_debug (1, "[%s] Command response error suppressed\n", PVT_ID(pvt));
```

Comment:
I'm not really interested in _that_ something is suppressed, but _what_ is suppressed here ;)

Changing error to debug should not _remove_ useful info.

(Also, please remove the spaces before function call parentheses everywhere.)